policyengine_us/parameters/gov/states/hi/tax/income/credits/non_refundable.yaml

```
@@ -0,0 +1,13 @@
+description: Hawaii provides the following non-refundable tax credits.
+values:
+  2021-01-01:
+    - hi_eitc
+  2023-01-01: 
```

Do we need this? Do we have a basic integration test to check?
```suggestion
  2023-01-01: {}
```

---

I dont think we need this, I believe Nikhil has added the feature of empty lists in yaml files, will add a test to double check

---

works but we need `[]`